roles/sap_ha_install_pacemaker/tasks/cluster_setup.yml

```diff
@@ -2,19 +2,37 @@
 # The following tasks are only included on the primary node!
 
 - name: "SAP Pacemaker Setup - Setup Pacemaker cluster"
```

Comment:
Command has been combined, name should be adjusted to `Setup and enable/start Pacemaker cluster`

Reply:
Updated title in https://github.com/sap-linuxlab/community.sap_install/commit/3f2319844d46743760384391e9b43ea1ab72be30